components/badges.css

```diff
@@ -0,0 +1,64 @@
+.badge-box {
+	position: relative;
+}
+.img-badge {
+	padding: 2px;
+	border-radius: 50%;
+	border: 2px solid white;
+	display: flex;
+	color: white;
+	justify-content: center;
+	align-items: center;
+	position: absolute;
+	font-size: 0.75em;
+	box-sizing: border-box;
+}
+.badge-online {
+	background-color: #00b300;
+}
+.badge-offline {
+	background-color: var(--color-grey-dark);
+}
+.badge-dnd {
+	background-color: red;
+}
+.icon-badge {
+	padding: 2px;
+	border-radius: 50%;
+	display: flex;
+	color: white;
+	justify-content: center;
+	align-items: center;
+	position: absolute;
+	background-color: var(--color-primary);
+}
+.badge-box .avatar-xs ~ .img-badge {
```

Comment:
What is this character here doing `~` ?

Reply:
It selects the siblings' element.
Here is the article I read: https://www.w3schools.com/css/css_combinators.asp